binary_search_tree/binary_search_tree.py

```diff
@@ -56,17 +56,41 @@ def for_each(self, cb):
     # Print all the values in order from low to high
     # Hint:  Use a recursive, depth first traversal
     def in_order_print(self, node):
-        pass
+        if self.left is not None:
+            self.left.in_order_print(self.left)
+        print(self.value)
```

Comment:
so, if there's nothing on the left, just print out the node that you are on right now, which if this is at the beginning of the "loop", would be the head